src/igvfd/types/institutional_certificate.py

```diff
@@ -28,3 +28,18 @@ class InstitutionalCertificate(Item):
 
     set_status_up = []
     set_status_down = []
+
+    @calculated_property(schema={
+        'title': 'Data Use Limitation Summary',
+        'type': 'string',
+        'description': 'A combination of the data use limitation and its modifiers',
+        'notSubmittable': True,
+        'uniqueKey': True
```

Comment:
Sorry - missed this earlier. I think this "uniqueKey" can probably also be dropped.

Reply:
OK, that makes sense actually. I saw it in other calculated properties but probably misinterpreted.